### PR TITLE
request https certificate from lets encrypt

### PR DESCRIPTION
**Pull request by jiegec**

### CODE REVIEW

File: lib/certificate-stack.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    readonly hostedZone: route53.IHostedZone;
+    readonly contactEmail: string;
+}
+
+export class CertificateStack extends cdk.NestedStack {
+    constructor(scope: cdk.Construct, id: string, props: CertificateProps) {
+        super(scope, id, props);
+
+        const domainName = props.hostedZone.zoneName;
+        const project = new codebuild.Project(this, `CertificateProject`, {
```
Comment (zxkane) [inline lib/certificate-stack.ts:20]: is it a job triggered by hand?

File: lib/certificate-stack.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import * as cdk from '@aws-cdk/core';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as sns from '@aws-cdk/aws-sns';
+import * as codebuild from '@aws-cdk/aws-codebuild';
+import * as iam from '@aws-cdk/aws-iam';
+import * as route53 from '@aws-cdk/aws-route53';
+
+export interface CertificateProps extends cdk.NestedStackProps {
+    readonly vpc: ec2.IVpc;
+    readonly notifyTopic: sns.ITopic;
```
Comment (zxkane) [inline lib/certificate-stack.ts:10]: let’s send the failure of build of certificate project to this topic

**Review by zxkane — CHANGES_REQUESTED** [review]

File: lib/certificate-stack.ts (hunk 1 above)
Comment (jiegec) [reply lib/certificate-stack.ts:20]: Yes.

File: lib/certificate-stack.ts (hunk 2 above)
Comment (jiegec) [reply lib/certificate-stack.ts:12]: Done.

**Review by zxkane — APPROVED** [review]

**Pull request merged**